src/diagnostics/conservation_diagnostics.jl

```diff
@@ -0,0 +1,143 @@
+# This file is included in Diagnostics.jl
+
+# Diagnostics for computing mass and energy conservation
+# All the diagnostics in this file are scalars, we need to
+# turn them into vectors as the diagnostics need to be mutable
```

Comment:
Can you open an issue in ClimaDiagnostics and link it here?

Reply:
Adde here: https://github.com/CliMA/ClimaDiagnostics.jl/issues/100